src/domain/wallet-invoices/index.types.d.ts

```diff
@@ -6,6 +6,10 @@ type WalletInvoice = {
   paid: boolean
 }
 
+type WalletInvoiceValidator = {
+  validateToSend({ fromWalletId: WalletId }): void | Error
+}
+
```

Comment:
I don't like having this as a void function. Should we make a `isValidToSend` that returns `true | Error`?

Reply:
We were discussing the same today. I think this is `true | Error` is the right answer given that two persons have noticed it.

There's a few other places with this pattern that I can change as well.

Reply:
Fixed here: https://github.com/GaloyMoney/galoy/pull/586/commits/ab6ace841bc33df931e0f3cd32c29a7ac5141baa

And also fixed in `LimitsChecker` and `TwoFA.verify` methods in #583 for consistency